### PR TITLE
Put longer lived Auth Cookie into preview

### PR DESCRIPTION
**Pull request by gklopper**
This is encrypted with the application secret.

At the moment people have to log in every hour.

### CODE REVIEW

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -114,7 +114,13 @@ object Switches {
   val EnableOauthOnPreview = Switch("Performance", "enable-oauth-on-preview",
     "If this switch is switched on then the preview server requires login",
     safeState = On,
-    sellByDate = new LocalDate(2015, 3, 31)
+    sellByDate = new LocalDate(2015, 4, 30)
```
Comment (gklopper) [inline common/app/conf/switches.scala]: Extending this switch while I am in here. There are some steps along the way to implementing PANDA and this is one of them.

File: standalone/app/controllers/OAuthLoginController.scala [hunk 2]
```diff
@@ -100,3 +106,26 @@ object OAuthLoginController extends Controller with ExecutionContexts with impli
     Redirect(routes.OAuthLoginController.login()).withNewSession
   }
 }
+
+object AuthCookie extends Logging {
+
+  private val cookieName = "GU_PV_AUTH"
+  private val thirtyDaysInSeconds: Int = 2592000
```
Comment (phamann) [inline standalone/app/controllers/OAuthLoginController.scala]: As discussed even 30days scares me from a security perspective. But seeing as its only preview i'm not that bothered...

**Comment by gklopper** [comment]
@phamann made it 1 day

**Comment by phamann** [comment]
:+1:

**Pull request merged**